### DISCUSSION TITLE
Getting 403 when using proxy

### DISCUSSION THREAD

**Discussion by BRV746** [quarkiverse/quarkus-cxf#683] [Q&A]
Hi, 

I get the error when using proxy for requests:
`com.ctc.wstx.exc.WstxIOException: Unable to tunnel through proxy. Proxy returns "HTTP/1.1 403 Forbidden"`

This is how I try to set up the proxy:
```
@PostConstruct
void configurePortType() {
        final HTTPConduit httpConduit = (HTTPConduit) ClientProxy.getClient(someSoap).getConduit();
        HTTPClientPolicy httpClientPolicy = new HTTPClientPolicy();
     
        httpClientPolicy.setConnectionTimeout(connectTimeout);
        httpClientPolicy.setReceiveTimeout(readTimeout);
        httpClientPolicy.setAllowChunking(false);
        httpClientPolicy.setProxyServer(proxyHost);
        httpClientPolicy.setProxyServerPort(proxyPort);

        AuthorizationPolicy authPolicy = new AuthorizationPolicy();
      
        authPolicy.setAuthorizationType(tokenType);
        authPolicy.setAuthorization(token);

        httpConduit.setAuthorization(authPolicy);
        httpConduit.setClient(httpClientPolicy);
    }
```

Maybe anyone could advise what could be the problem? No authorization is needed for proxy and I had set `jdk.http.auth.tunneling.disabledSchemes=Basic` but it did not help.

**Comment by shumonsharif** [comment]
> Maybe anyone could advise what could be the problem? No authorization is needed for proxy and I had set `jdk.http.auth.tunneling.disabledSchemes=Basic` but it did not help.

Hi @BRV746 If no authorization is needed for the proxy, why are you setting an `AuthorizationPolicy`? 

I would suggest you try some simpler code just to test and troubleshoot connectivity issues.  Perhaps something like the below may work, though this is completely untested, as I do not have a proxy to try against.

```java
String https_url = "https://someSoap.wsdl.url";
Proxy proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress(proxyHost, proxyPort));
URL url = new URL(https_url);
HttpsURLConnection con = (HttpsURLConnection)url.openConnection(proxy);
System.out.println("Response-" + con.getResponseCode());   // Should return 200
```
